src/components/menu-top/MenuTop.vue

```diff
@@ -20,6 +20,13 @@
 
           <hr class="navbar-divider">
 
+          <b-navbar-item
+            data-test="menu-top--save-show"
+            @click="saveFile()"
+          >
+            Save Show
```

Comment:
Instead of "Save Show" and "Load Show", we should use "Export Show" and "Import Show". Since this is a web application, it will "download" the show file to the local computer, not save it.